packages/integrations/cloudflare/src/entrypoints/server.ts

```diff
@@ -6,8 +6,11 @@ import type {
 import type { SSRManifest } from 'astro';
 import { App } from 'astro/app';
 import { setGetEnv } from 'astro/env/setup';
+import { env as globalEnv } from 'cloudflare:workers';
 import { createGetEnv } from '../utils/env.js';
 
+setGetEnv(createGetEnv(globalEnv as any));
```

Comment:
Could the argument for `createGetEnv` be changed, rather than casting here?

Reply:
No it cannot because it's Cloudflare.Env is {} unfortunately